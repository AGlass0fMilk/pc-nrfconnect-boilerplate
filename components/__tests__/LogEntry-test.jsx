import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateLogEntry,
} from '../LogEntry';

describe('LogEntry', () => {
    it('should implement component decorator', () => {
        expect(decorateLogEntry).toBeInstanceOf(Function);
    });
});

describe('decorateLogEntry', () => {
    it('should render component', () => {
        const LogEntry = decorateLogEntry('div');
        expect(renderer.create(
            <LogEntry />,
        )).toMatchSnapshot();
    });
});
