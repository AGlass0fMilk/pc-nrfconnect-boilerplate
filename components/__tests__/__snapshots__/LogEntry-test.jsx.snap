exports[`decorateLogEntry should render component 1`] = `<div />`;
